### ISSUE TITLE
Add/Subtract from Selection acting weird

### ISSUE THREAD

**Issue by Remixful** [aseprite/aseprite#1626]
Describe your bug report or feature request here
Attempting to add/subtract (SHIFT/ALT-SHIFT) to selection makes the selection size a fixed ratio. I assume it's due to the shift keys being used. On top of that, it's very uncomfortable to use ALT-SHIFT since it keeps going to eyedropper first sometimes.

![8dcb3c2914b34bae79e98dbf7e7ade1b](https://user-images.githubusercontent.com/8620268/33723483-d17763d6-db3a-11e7-830e-378c6b2ab3b2.gif)

### Aseprite and System version
Latest Steam version (v1.2.4-x64)
Windows 10

**Comment by dacap** [comment]
Yes, there are several keys competing when we use the selection:

1. With the Shift key we can switch to the "Add" selection mode, we press the mouse button, and then the Shift key starts acting to select a square. The solution at this moment is releasing the Shift key after we used it initially to start the "Add" selection mode. So: Press Shift, press mouse button, release Shift key, move the mouse, and then release the mouse button.
1. The Alt+Shift is a little tricky, to select the subtraction mode we have to press Shift key, and then Alt key. If we press Alt key first, the eyedropper tool is selected (which can be useful to change the background color used in a cut operation, there were an issue where this was discussed, I cannot remember). So: Press Shift, then press Alt, press the mouse button, now you are in Subtraction mode (if you press Alt key first, the Eyedropper is selected). As an alternative method, you can use the right click to subtract selection areas.

I'm still not sure how to solve these UI problems.
